<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DanmakuMenu</class>
 <widget class="QWidget" name="DanmakuMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>116</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="minimumSize">
      <size>
       <width>145</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>145</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:none;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:none;</string>
        </property>
        <property name="text">
         <string>danmaku display</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="styleSheet">
           <string notr="true">background:transparent;
border:none;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="bigFontRadioButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>26</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	image:url(:/icon/big.png);
}

QRadioButton::indicator:checked {
	image:url(:/icon/big_select.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">fontButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="smallRadioButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>26</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	image:url(:/icon/small.png);
}

QRadioButton::indicator:checked {
	image:url(:/icon/small_select.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">fontButtonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="minimumSize">
             <size>
              <width>58</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>58</width>
              <height>26</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
width:58px;
height:26px;
background:rgba(103,108,117,1);
border-radius:13px;
}

QPushButton:hover{
width:58px;
height:26px;
background:rgba(136,143,154,1);
border-radius:13px;
}
</string>
            </property>
            <property name="text">
             <string>reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>129</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>129</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;
border:none;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:none;</string>
        </property>
        <property name="text">
         <string>danmaku color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="styleSheet">
           <string notr="true">background:transparent;
border:none;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="whiteRadioButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	width:16px;
	height: 16px;
	background-color: #FFFFFFFF;
	border:2px solid transparent;
	border-radius:10px;
}

QRadioButton::indicator:checked {
	width:16px;
	height: 16px;
	border:2px solid #FF00B1EA;
	border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="redRadioButton">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	width:16px;
	height: 16px;
	background-color: #FFE70012;
	border:2px solid transparent;
	border-radius:10px;
}

QRadioButton::indicator:checked {
	width:16px;
	height: 16px;
	border:2px solid #FF00B1EA;
	border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="yellowRadioButton">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	width:16px;
	height: 16px;
	background-color: #FFFEF103;
	border:2px solid transparent;
	border-radius:10px;
}

QRadioButton::indicator:checked {
	width:16px;
	height: 16px;
	border:2px solid #FF00B1EA;
	border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="greenRadioButton">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	width:16px;
	height: 16px;
	background-color: #FF009843;
	border:2px solid transparent;
	border-radius:10px;
}

QRadioButton::indicator:checked {
	width:16px;
	height: 16px;
	border:2px solid #FF00B1EA;
	border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorButtonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="styleSheet">
           <string notr="true">background:transparent;
border:none;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QRadioButton" name="cyanRadioButton">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	width:16px;
	height: 16px;
	background-color:#FF00A0EA;
	border:2px solid transparent;
	border-radius:10px;
}

QRadioButton::indicator:checked {
	width:16px;
	height: 16px;
	border:2px solid #FF00B1EA;
	border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="magentaRadioButton">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	width:16px;
	height: 16px;
	background-color: #FFE2027F;
	border:2px solid transparent;
	border-radius:10px;
}

QRadioButton::indicator:checked {
	width:16px;
	height: 16px;
	border:2px solid #FF00B1EA;
	border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="drakGreenRadioButton">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	width:16px;
	height: 16px;
	background-color: #FF002E72;
	border:2px solid transparent;
	border-radius:10px;
}

QRadioButton::indicator:checked {
	width:16px;
	height: 16px;
	border:2px solid #FF00B1EA;
	border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorButtonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="grayRadioButton">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QRadioButton::indicator {
	width:16px;
	height: 16px;
	background-color: #FF683A7B;
	border:2px solid transparent;
	border-radius:10px;
}

QRadioButton::indicator:checked {
	width:16px;
	height: 16px;
	border:2px solid #FF00B1EA;
	border-radius:10px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorButtonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource/player.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="fontButtonGroup"/>
  <buttongroup name="colorButtonGroup"/>
 </buttongroups>
</ui>
